bicep/main.bicep

```diff
@@ -1514,6 +1517,17 @@ resource aks_law 'Microsoft.OperationalInsights/workspaces@2022-10-01' = if (cre
   )
 }
 
+
+resource containerLogsV2_Basiclogs 'Microsoft.OperationalInsights/workspaces/tables@2022-10-01' = if(containerLogsV2BasicLogs){
+  name: '${aks_law_name}/ContainerLogV2'
+  properties: {
+    plan: 'Basic'
+  }
+  dependsOn: [
```

Comment:
Do we need the log analytics table creation to wait for the AKS cluster?
I'd hope it's not connected, and we can drop the dependsOn.